### PR TITLE
Add property for source transaction

### PR DESCRIPTION
**Pull request by cwoolum**
Added source_transaction property to facilitate Special Case Transfers for Stripe connect.

Related ticket #333

### CODE REVIEW

File: src/Stripe/Services/Transfers/StripeTransferCreateOptions.cs [hunk 1]
```diff
@@ -26,6 +26,12 @@ public class StripeTransferCreateOptions
         [JsonProperty("statement_descriptor")]
         public string StatementDescriptor { get; set; }
 
+        /// <summary>
+        /// You can use this parameter to transfer funds from a charge (or other transaction) before they are added to your available balance. A pending balance will transfer immediately but the funds will not become available until the original charge becomes available. See the Connect documentation for details.
+        /// </summary>
+        [JsonProperty("source_transaction")]
+        public string SourceTransaction { get; set; }
```
Comment (jaymedavis) [inline src/Stripe/Services/Transfers/StripeTransferCreateOptions.cs]: How do you know this works?

**Comment by jaymedavis** [comment]
This actually looks like it belongs on the StripeTransfer, and not the StripeTransferCreateOptions. I will add it there, this doesn't look like something you set.

**Comment by cwoolum** [comment]
No, it's definitely on create. For Stripe connection, you can't have more than 10% of your transfers be withour a source transaction for compliance reasons

https://stripe.com/docs/api#create_transfer

**Comment by cwoolum** [comment]
Also see https://stripe.com/docs/connect/special-case-transfers#associating-a-transfer-with-a-charge

**Comment by jaymedavis** [comment]
thanks :+1:

**Pull request closed without merging**